metrics/go_metrics_test.go

```diff
@@ -83,7 +83,6 @@ func TestNewMetrics(t *testing.T) {
 	ensureContains(t, registry, "request_over_head_time.pre-bidder", m.OverheadTimer[PreBidder])
 	ensureContains(t, registry, "request_over_head_time.make-auction-response", m.OverheadTimer[MakeAuctionResponse])
 	ensureContains(t, registry, "request_over_head_time.make-bidder-requests", m.OverheadTimer[MakeBidderRequests])
-	ensureContains(t, registry, "request_over_head_time.bidder-server-response", m.OverheadTimer[BidderServerResponse])
```

Comment:
should assert `bidder-server-response` 
```
ensureContains(t, registry, "bidder-server-response", m.BidderServerResponseTimer)
```

Reply:
addressed in [e1b7872](https://github.com/prebid/prebid-server/pull/2722/commits/e1b787255ead29525c012a8d2eb4767da98b5bf5)